pontoon/base/static/css/style.css

```diff
@@ -21,6 +21,10 @@
   content: "";
 }
 
+.missing .status.fa:before {
+  color:#5F7285;
+}
+
```

Comment:
Same as in `Entity.css`: no need to add a new rule, just change the color on `.status.fa:before {` to `#5F7285`.

Please also search for `.missing` within the `style.css` file and change background and border values to `5F7285`. I'm not exactly sure these rules are still being used, but let's at least keep the values consistent.